ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -97,54 +91,48 @@ const CancelSpeedupPopover = () => {
     >
       <AppLoadingSpinner className="cancel-speedup-popover__spinner" />
       <div className="cancel-speedup-popover__wrapper">
-        <Typography
-          boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
-          variant={TypographyVariant.H6}
-          marginTop={0}
+        <Text
+          variant={TextVariant.bodySm}
+          className="cancel-speedup-popover__description"
+          // gets passed through to Box component
+          display={DISPLAY.FLEX}
           marginBottom={2}
+          paddingBottom={2}
         >
           {t('cancelSpeedUpLabel', [
-            <strong key="cancelSpeedupReplace">{t('replace')}</strong>,
+            <Text as="strong" key="cancelSpeedupCancel">
+              {t('replace')}
+            </Text>,
```

Comment:
I was wondering why the `<strong>` tag wasn't working and it's because it's wrapped in a `span` which must be created by the local function. Maybe we should update the css again so no matter what levels of nesting strong tags are always bolded.

I think we still need to add the same variant here though so it retains the same font-size
```suggestion
            <Text as="strong" variant={TextVariant.bodySm} key="cancelSpeedupCancel">
              {t('replace')}
            </Text>,
```

Reply:
added the `variant`. For me the boldening is actually working, albeit unexpectedly because of some different css rule for `.mm-text--font-weight-bold` somewhere in code base:

<img width="758" alt="Screenshot 2023-04-28 at 11 21 30" src="https://user-images.githubusercontent.com/90316/235109791-48da2083-a56d-46cc-858a-3a7efef2e24a.png">